lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -1939,16 +1939,8 @@ defmodule LightningWeb.WorkflowLive.Edit do
     end
   end
 
-  defp webhook_url(trigger) do
-    trigger
-    |> case do
-      %{type: :webhook, id: id} ->
-        Routes.webhooks_url(LightningWeb.Endpoint, :create, [id])
-
-      _ ->
-        nil
-    end
-  end
+  defp webhook_url(%{id: id}),
```

Comment:
Given that verified routes are currently favoured over route helpers, could you update this to use [url/1](https://hexdocs.pm/phoenix/Phoenix.VerifiedRoutes.html#url/1) instead?

Reply:
Done